samples/petclinic-mysql.yaml

```diff
@@ -46,10 +46,50 @@ metadata:
   name: mysql-binding
 type: Opaque
 stringData:
-  hostname: localhost
-  jdbcUrl: jdbc:mysql://localhost:3306/default?user=root&password=
+  hostname: mysql
+  jdbcUrl: jdbc:mysql://mysql:3306/default?user=root&password=root
```

Comment:
The extra data doesn't hurt but FWIW, this would work with just `jdbcUrl` in the secret data

Reply:
not every app in this world is a java app, although petclinic is

I got the fields by cribbing from VCAP_SERVICES